provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/resources/Practitioner.java

```diff
@@ -0,0 +1,229 @@
+package gov.va.api.health.providerdirectory.api.resources;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import gov.va.api.health.providerdirectory.api.Fhir;
+import gov.va.api.health.providerdirectory.api.bundle.AbstractBundle;
+import gov.va.api.health.providerdirectory.api.bundle.AbstractEntry;
+import gov.va.api.health.providerdirectory.api.bundle.BundleLink;
+import gov.va.api.health.providerdirectory.api.datatypes.Address;
+import gov.va.api.health.providerdirectory.api.datatypes.Attachment;
+import gov.va.api.health.providerdirectory.api.datatypes.CodeableConcept;
+import gov.va.api.health.providerdirectory.api.datatypes.ContactPoint;
+import gov.va.api.health.providerdirectory.api.datatypes.HumanName;
+import gov.va.api.health.providerdirectory.api.datatypes.Identifier;
+import gov.va.api.health.providerdirectory.api.datatypes.Period;
+import gov.va.api.health.providerdirectory.api.datatypes.Signature;
+import gov.va.api.health.providerdirectory.api.datatypes.SimpleResource;
+import gov.va.api.health.providerdirectory.api.elements.BackboneElement;
+import gov.va.api.health.providerdirectory.api.elements.Element;
+import gov.va.api.health.providerdirectory.api.elements.Extension;
+import gov.va.api.health.providerdirectory.api.elements.Meta;
+import gov.va.api.health.providerdirectory.api.elements.Narrative;
+import gov.va.api.health.providerdirectory.api.elements.Reference;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+@Schema(
+  description = "http://hl7.org/fhir/practitioner.html"
+)
+public class Practitioner implements DomainResource {
+  @NotBlank String resourceType;
+
+  @Pattern(regexp = Fhir.ID)
+  String id;
+
+  @Valid Meta meta;
+
+  @Pattern(regexp = Fhir.URI)
+  String implicitRules;
+
+  @Pattern(regexp = Fhir.CODE)
+  String language;
+
+  @Valid Narrative text;
+
+  @Valid List<SimpleResource> contained;
+
+  @Valid List<Extension> extension;
+
+  @Valid List<Extension> modifierExtension;
+
+  @NotNull @Valid List<PractitionerIdentifier> identifier;
+
+  boolean active;
+
+  @Valid PractitionerHumanName name;
+
+  @Valid List<ContactPoint> telecom;
+
+  @Valid List<Address> address;
+
+  Gender gender;
+
+  @Pattern(regexp = Fhir.DATE)
+  String birthDate;
+
+  @Valid List<Attachment> photo;
+
+  @Valid List<PractitionerRole> practitionerRole;
```

Comment:
Not seeing this field on the spec.

http://www.fhir.org/guides/argonaut/pd/StructureDefinition-argo-practitioner.html